monitoring/dashboard.py

```diff
@@ -525,6 +542,111 @@ def top10_errors_by_bucket(title, code):
     title="5xx : Top10 by Bucket", code='~"5.."'
 )
 
+
+quotaHealth = GaugePanel(
+    title="Quota service uptime",
```

Comment:
It's a little weird to show a gauge for the `mean` during `__rate_interval`

You could see more info if you graph in a time series to see over time when it was up or down and also know at the current time if it's up or down without a `mean`

Reply:
Updated with on/off display over time:

![image](https://github.com/scality/cloudserver/assets/91462779/a767538f-19b2-48ec-bfc9-23bf527d7b08)

Not related, but FYI, I also added another graph, with the utilization metrics retrieval latencies, I was not using the metric before, while it's useful to see the impact of the internal calls in during the quota evaluation logic:

![image](https://github.com/scality/cloudserver/assets/91462779/99c7c7de-9fb5-4eee-93f2-411a0617ec21)
(the one in the middle, bottom)

Reply:
About On / Off, I think you have a percentage with your request, so you could keep a percentage to see if you have some instances that can't connect

Reply:
And for the metric retrieval latencies, should you really have a big diff between this and the quota evaluation ? Also you only record success for metric retriaval

Reply:
Yes, metric retrieval can be done twice in an API call, in case we need to cleanup when quota is exceeded. We still want to be able the latency with scuba, independently from the quota evaluation logic

(please feel free to challenge that)

Reply:
I think the utilization metric retrieval metric is the same as `scuba_http_request_duration_seconds{action="GetMetrics"}`
(https://github.com/scality/scuba/pull/104/files#diff-7c16f2a12075d20e3640e31bcb11cc1b87aad07a0fc048ace705020732a5a2cfR21-R28) but with network time over it.

But then maybe scuba metrics can record request that come from other places than cloudserver, so maybe you should actually keep this

___
Otherwise I posted in another comment, it could be nice to have another scuba endpoint to aggregate all the requests into one, especially if it's a code path that's called often.

Reply:
> But then maybe scuba metrics can record request that come from other places than cloudserver, so maybe you should actually keep this

Yes it's indeed the same, but this is useful to see the impact of internal network communications, as seen by S3, as the scuba metrics may be, in the future, including public ones with authz, expected to be slower...

For the other endpoint, yes, see https://scality.atlassian.net/browse/ZENKO-4801 (the current epic supports account, but the use case is bucket)